internal/services/v1/experimental_test.go

```diff
@@ -435,10 +434,9 @@ func TestBulkCheckPermission(t *testing.T) {
 	defer cleanup()
 
 	testCases := []struct {
-		name                  string
-		requests              []string
-		response              []bulkCheckTest
-		expectedDispatchCount int
+		name     string
+		requests []string
+		response []bulkCheckTest
```

Comment:
why is the dispatch count gone?

Reply:
We removed the caveats so now the dispatch count varies depending on which branch executes first, causing the test to become flaky. Since we have a dispatch count test elsewhere, I removed it here